README.md

```diff
@@ -183,6 +183,69 @@ NB: It's also possible to run qdl from the host while the baord is not connected
 
 Want to join in the development? Changes welcome! See [CONTRIBUTING.md file](CONTRIBUTING.md) for step by step instructions.
 
+### Test an image locally with qemu
+
+You can boot an image locally with qemu as follows:
+
+1. As above under "Usage", build the disk image from the root filesystem
+   tarball if you haven't done this already:
+    ```bash
+    debos debos-recipes/qualcomm-linux-debian-image.yaml
+    ```
+
+1. Run qemu as follows:
+    ```bash
+    # SCSI is required to present a device with a matching 4096 sector size
+    # inside the VM
+    qemu-system-aarch64 -cpu cortex-a57 -m 2048 -M virt -nographic \
```

Comment:
That made me wonder how we should define the QEMU model. We could pick it to be fast and manage it like a reference device that we maintain, or we could try to follow the most constrained environment we're trying to support. For the latter, the RB1 is probably the oldest board we're trying to enable, so 2GiB of RAM is fitting. I don't think QEMU has support for Kryo CPUs, so not sure what Cortex equivalent we should pick.